tests/whitelist.txt

```diff
@@ -4,9 +4,6 @@ integration.fileserver.test_fileclient
 integration.fileserver.test_roots
 integration.loader.test_ext_grains
 integration.loader.test_ext_modules
-integration.loader.test_globals
-integration.loader.test_interfaces
-integration.loader.test_loader
```

Comment:
These are now `unit.loader.<same module name>`. Unless you're leaving unit tests for last.